src/olympia/files/utils.py

```diff
@@ -314,7 +316,16 @@ def apps(self):
                 yield Extractor.App(
                     appdata=app, id=app.id, min=min_appver, max=max_appver)
             except AppVersion.DoesNotExist:
-                pass
+                couldnt_find_version = True
+
+        specified_versions = self.strict_min_version or self.strict_max_version
+
+        if couldnt_find_version and specified_versions:
+            msg = _(
+                'Cannot find min/max version. Maybe '
+                '"strict_min_version" or "strict_max_version" '
+                'contains a not supported version?')
```

Comment:
Wording seems a bit unpleasant to read. Replace the end with `an unsupported version` maybe ?